src/Symfony/Component/DependencyInjection/Compiler/AutowireRequiredMethodsPass.php

```diff
@@ -67,6 +69,7 @@ protected function processValue(mixed $value, bool $isRoot = false): mixed
                     if (false === stripos($doc, '@inheritdoc') || !preg_match('#(?:^/\*\*|\n\s*+\*)\s*+(?:\{@inheritdoc\}|@inheritdoc)(?:\s|\*/$)#i', $doc)) {
                         break;
                     }
+                    trigger_deprecation('symfony/dependency-injection', '6.3', 'The "@required" annotation on methods is deprecated, use the "Symfony\Contracts\Service\Attribute\Required" attribute instead.');
```

Comment:
this one looks wrong to me. It will trigger the deprecation for any method using `@inheritdoc`, which is totally wrong

Reply:
I had the same question: https://github.com/symfony/symfony/pull/48990#issuecomment-1408394474